### DISCUSSION TITLE
Reorganization of the Theme Settings

### DISCUSSION THREAD

**Discussion by pryley** [AnubisNekhet/AnuPpuccin#59] [Ideas]
@AnubisNekhet 

I think it would be nice to organize and simplify the theme settings a little bit so they are easier to navigate.

This is what I have come up with so far, but it's just an idea.

Thoughts?

## Theme Settings (collapsed):

![collapsed](https://user-images.githubusercontent.com/134939/215288730-8fdc0e40-ebee-418a-94ed-2be8ab83b5ba.png)

## Theme Settings (expanded):
 
![expanded](https://user-images.githubusercontent.com/134939/215288732-9ca9921d-8ffc-4145-9c11-28a17256976b.png)

## Text Inputs for theme links

Since you cannot add links to the theme settings, I have used text inputs which can be copy/pasted into a browser. And while they _could_ be overwritten, there is always a button to reset the value back to the default link.

<img width="714" alt="image" src="https://user-images.githubusercontent.com/134939/215284776-b2990cee-5ce8-4222-aec6-d007daaef10b.png">

<img width="698" alt="image" src="https://user-images.githubusercontent.com/134939/215284827-cc5b4675-6603-40f4-9879-dcb3cfdd6886.png">

## Unit Hints for Number Input Fields

<img width="663" alt="image" src="https://user-images.githubusercontent.com/134939/215284795-80880636-8fc8-43da-88d4-720541670c64.png">

**Comment by AnubisNekhet** [comment]
@pryley Style Settings has released an info-text field that supports markdown so I'll be changing links to match that format. Just letting you know in advance.

**Reply by pryley** [reply]
Nice!
